apps/functions-subscription/src/adapters/azure/functions/__tests__/get-subscription.test.ts

```diff
@@ -106,6 +106,60 @@ describe('makeGetSubscriptionHandler', () => {
       makeFunctionContext(),
     );
 
+    expect(actual.status).toStrictEqual(200);
+    expect(await actual.json()).toMatchObject({
+      userId: aSubscription.userId,
+      trialId: aSubscription.trialId,
+      state: aSubscription.state,
+    });
+  });
+  it('should return 200 when the subscription exist and the x-user-groups header has ApiTrialManager', async () => {
+    const env = makeTestSystemEnv();
+
+    env.getSubscription.mockReturnValueOnce(TE.right(aSubscription));
+
+    const request = new HttpRequest({
+      url: makeAValidGetSubscriptionRequest().url,
+      params: makeAValidGetSubscriptionRequest().params,
+      method: makeAValidGetSubscriptionRequest().method,
+      headers: {
+        ...managerHttpRequestHeaders,
+        'x-user-groups': 'ApiTrialManager',
+      },
+    });
+
+    const actual = await makeGetSubscriptionHandler(env)(
+      request,
+      makeFunctionContext(),
+    );
+
+    expect(actual.status).toStrictEqual(200);
+    expect(await actual.json()).toMatchObject({
+      userId: aSubscription.userId,
+      trialId: aSubscription.trialId,
+      state: aSubscription.state,
+    });
+  });
```

Comment:
**question**: do you think we need this unit test?

What is this unit test testing that is not tested with [this unit test](https://github.com/pagopa/trial-system/blob/53d0e418a5177405cdb8fa2113a382a0329608d8/apps/functions-subscription/src/adapters/azure/functions/__tests__/get-subscription.test.ts#L71-L87)?

Reply:
Removed in [ffc2d07](https://github.com/pagopa/trial-system/pull/256/commits/ffc2d0738cc7ad555c148b3ba8d571e92aafa983)